pkg/plugins/backendplugin/coreplugin/registry.go

```diff
@@ -254,7 +254,7 @@ func NewPlugin(pluginID string, cfg *setting.Cfg, httpClientProvider *httpclient
 	if backendFactory == nil {
 		return nil, ErrCorePluginNotFound
 	}
-	bp, err := backendFactory(p.ID, p.Logger(), nil)
+	bp, err := backendFactory(p.ID, p.Logger(), p.Tracer(), nil)
```

Comment:
Think this simplifies, but I might misunderstand why plugin needs to hold a tracer?

```suggestion
	bp, err := backendFactory(p.ID, p.Logger(), tracer, nil)
```

Reply:
I think services do not pass log explicitly, they pass/return plugin usually through reference and get the log.

Reply:
Similar to https://github.com/grafana/grafana/blob/main/pkg/tsdb/loki/loki.go#L117 so we don't have to change the signature of `CallResource` method that requires multiple [changes](https://github.com/grafana/grafana/blob/main/pkg/tsdb/loki/loki.go#L110).